src/engine/Docs/Actors.md

```diff
@@ -0,0 +1,253 @@
+## Basic actors
+
+For quick and dirty games, you can just create an instance of an `Actor`
+and manipulate it directly.
+
+Actors (and other entities) must be added to a [[Scene]] to be drawn
+and updated on-screen.
+
+```ts
+var player = new ex.Actor();
+
+// move the player
+player.dx = 5;
```

Comment:
I know this wasn't part of your issue, so if we want to make a separate on for docs clean-up let me know.

This should be `player.vel.x`, the old way `player.dx` has been deprecated.